src/components/AssetsTable/index.jsx

```diff
@@ -197,6 +196,38 @@ export class AssetsTable extends React.Component {
     };
   }
 
+  getUploadStatusFields(assetsStatus) {
+    switch (assetsStatus.type) {
+      case assetActions.UPLOAD_ASSET_SUCCESS:
+        return {
+          alertType: 'success',
+          alertDialog: `${assetsStatus.loadedCount} files successfully uploaded.`,
+        };
+      case assetActions.UPLOADING_ASSETS:
+        return {
+          alertType: 'info',
+          alertDialog: `${assetsStatus.count} files uploading.`,
```

Comment:
I like this idea! I'll double check w/ docs on this.

Reply:
@srpearce -- I used your copy in the files & uploads status message when files are uploading (see image attached). What do you think about adding ellipsis at the end of the message?

![screen shot 2018-01-04 at 10 33 20 am](https://user-images.githubusercontent.com/5265058/34570856-ffe2e220-f13a-11e7-9faf-14545c57bf62.png)

Reply:
I like the idea in theory—the only problem is i18n. Some languages may not have a comparable punctuation mark. Because of that, I think it would be more effective to leave the period and add a spinner.